src/Components/Search.jsx

```diff
@@ -0,0 +1,23 @@
+import {Divider, Input} from 'antd'
+import {useState} from 'react';
+
+function Search(props) {
+  let [searchFood, setSearchFood] = useState("");
+
+  let handleSelect = (e) => {
+    let inputSearch = e.target.value.toLowerCase()
+    setSearchFood(inputSearch);
+    props.search(inputSearch);
```

Comment:
the props that you're passing to this component is called `searchFood` and not `search`